Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Beam/RegistrationToken.hs

```diff
@@ -33,6 +33,7 @@ data RegistrationTokenT f = RegistrationTokenT
     tokenExpiry :: B.C f Int,
     entityId :: B.C f Text,
     merchantId :: B.C f Text,
+    merchantOperatingCityId :: B.C f Text,
```

Comment:
You didn't migration for this in sql file, and this should also be Maybe like Booking, SearchRequest tables else it will fail for older registration token.

Reply:
Done